modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -380,12 +388,555 @@ export default class Ecdsa {
       proof: proofShare,
     };
 
-    return signers;
+    return {
+      wShare,
+      kShare,
+    };
+  }
+
+  /**
+   * Perform multiplicitive-to-additive (MtA) share conversion with another signer.
+   * Connection 1.2 in https://lucid.app/lucidchart/7061785b-bc5c-4002-b546-3f4a3612fc62/edit?page=IAVmvYO4FvKc#
+   * @param {SignConvert} shares
+   * @returns {SignConvertRT}
+   */
+  async signConvertStep1(shares: SignConvertStep1): Promise<SignConvertStep1Response> {
```

Comment:
Note: these methods can be further optimized in future PR's. 

This PR just splits up the behemoth that is signConvert, without optimizing those functions further.